### DISCUSSION TITLE
Maybe a solution?

### DISCUSSION THREAD

**Discussion by JA16122000** [mrlt8/docker-wyze-bridge#787] [Ideas]
I was thinking @mrlt8 , why not reconnect every 15 minutes to record? 
Now this goes on indefinitely, with the result that after a while, the recordings become shorter than 15 minutes. 

Automatically reconnecting each time and thus starting a 15-minute recording again will fix the following problem I suspect:
-	No long-term problems with night/day speed difference (FPS) because a new connection is established every 15 minutes #749 #609     #540
-	A fresh connection will ensure a stable 15-minute recording without loss of time. #623 #722 #540

**Comment by mrlt8** [comment]
Could potentially cause more issues as reconnects could take a while and potentially fail.

I did add some [controls](https://github.com/mrlt8/docker-wyze-bridge/blob/47d2416aebdd7eaae4b2ea2ae35eddfeb1b8d226/app/wyzecam/tutk/tutk_protocol.py#L256-L280) to start/stop the audio/video independently while maintaining the connection which could be a potential solution, but it hasn't been fully implemented in the bridge yet.

I have also been looking into tweaking the ffmpeg command, but it seems as though the problem is most likely coming from the way the camera is encoding the video and would probably require the video to be re-encoded. `FORCE_ENCODE=true` would force the video to be re-encoded, though it will require more processing power to do so.
